challenges/objects-arrays.js

```diff
@@ -50,21 +80,32 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+for (i=0; i < graduates.length; i++) {
+  universities.push(graduates[i]["university"])
 console.log(universities)
-
+}
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (i=0; i < graduates.length; i++) {
+  contactInfo.push(graduates[i]["first_name"]);
```

Comment:
That solution works, but it will generate a coma `,` for You and push emai to the new line.